src/config.webpack.js

```diff
@@ -254,15 +268,19 @@ module.exports = function ({ runtime }) {
       ],
     },
     resolve: {
-      modules               : moduleDirs,
-      descriptionFiles      : ['package.json'],
-      mainFields            : ['browser', 'main'],
-      mainFiles             : ['index'],
-      aliasFields           : ['browser'],
-      extensions            : ['.js'],
-      enforceExtension      : false,
-      enforceModuleExtension: false,
-      alias                 : runtime.alias,
+      modules         : moduleDirs,
+      descriptionFiles: ['package.json'],
+      mainFields      : ['browser', 'main'],
+      mainFiles       : ['index'],
+      aliasFields     : ['browser'],
+      extensions      : ['.js'],
+      enforceExtension: false,
+      fallback        : {
+        stream: require.resolve('stream-browserify'),
```

Comment:
I wonder what's requiring that, it's a bit unusual for client side modules to depend on node.js style streams in the year 202x 🤔 probably that would need to be addressed in the content repo (if at all, it's a reliable module so there's no real reason to get rid of it aside from its modest bundled size)

Reply:
I checked and it was:

```
❯ npm ls imsc
content@0.0.1 /home/kvz/code/content
└─┬ dashjs@3.2.1 invalid
  └── imsc@1.1.2  extraneous
```

I upgraded it to the latest version, but it remained the same, that's as far as I researched it. I think it did it before too, so at least we won't regress on it, but it would be nice to improve later likely :ok_hand: